evalml/pipelines/components/estimators/clusterers/dbscan.py

```diff
@@ -0,0 +1,54 @@
+"""DBSCAN Clusterer."""
+from sklearn.cluster import DBSCAN as SKDBSCAN
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+from evalml.utils import infer_feature_types
+
+
+class DBSCANClusterer(Estimator):
```

Comment:
Just FYI, dbscan only finds one cluster for the breast cancer dataset with and without scaling:

![image](https://user-images.githubusercontent.com/41651716/158895608-07d9b1be-4a40-4087-a9db-d359c1efc6e2.png)

Reply:
I'm not surprised. Of all these initial models, DBSCAN is most sensitive to hyperparameter values. Once we have a good way to performance test the models I'd love to adjust the default values, but for now I think following the sklearn default values makes the most sense.